shared/actions/chat2/index.js

```diff
@@ -715,7 +715,7 @@ const loadMoreMessages = (
         messageTypes: loadThreadMessageTypes,
       },
       reason:
-        action.type.reason === 'push'
+        reason === 'push'
```

Comment:
I noticed in the logs I wasn't seeing this, I guess this is the fix?

Reply:
yeah